pages/games/[hash].js

```diff
@@ -0,0 +1,52 @@
+import styles from "../../styles/Game.module.css";
+
+function Game({ data }) {
+  return (
+    <div>
+      Game page {data.game_hash}
+      <div className={styles.board}>
+        <div className={styles.fleet}>
+          {data.ships.map((ship, i) => (
+            <div
+              key={i}
```

Comment:
It's good practice to use a unique key derived from the ship rather than its index.
This way react can tract the element even if its position change in the array.

Reply:
Yes, I knew that. Was just too lazy. Going to add it :+1: